pkg/es/config/config.go

```diff
@@ -39,15 +39,26 @@ const (
 
 // IndexOptions describes the index format and rollover frequency
 type IndexOptions struct {
-	Priority          int64  `mapstructure:"priority"`
-	DateLayout        string `mapstructure:"date_layout"`
-	Shards            int64  `mapstructure:"shards"`
-	Replicas          int64  `mapstructure:"replicas"`
+	// Priority contains the of index template (ESv8 only).
+	Priority   int64  `mapstructure:"priority"`
+	DateLayout string `mapstructure:"date_layout"`
+	// Shards is the number of shards per index in Elasticsearch.
+	Shards int64 `mapstructure:"shards"`
+	// Replicas is the number of replicas per index in Elasticsearch.
+	Replicas int64 `mapstructure:"replicas"`
+	// RolloverFrequency rotates indices over the given period. For example, "day" creates
+	// "jaeger-{index_type}-yyyy-MM-dd" every day after UTC 12AM.
+	// Valid configuration options are: [hour, day].
+	// This does not delete old indices.
+	// For details on complete index management solutions supported by Jaeger,
+	// refer to: https://www.jaegertracing.io/docs/deployment/#elasticsearch-rollover
```

Comment:
Link does not appear to be valid

Reply:
@yurishkuro do you mean that the link doesn't work or that its pointing to the wrong thing?

Reply:
hm, looks like we have redirects - typically there should be a version after docs/ (with `latest` being a valid version, also via redirects). I am just not sure this link will survive once we release v2 - there's definitely going to be /deployment/, but we plan to move storage info to subpages.

Problem for another day.